tgui/packages/tgui/interfaces/PersonalCrafting.tsx

```diff
@@ -852,6 +868,25 @@ const RecipeContent = ({ item, craftable, busy, mode, diet }) => {
                   }
                 />
               )}
+              {!item.non_craftable && item.mass_craftable && (
+                <Button
+                  width="30px"
+                  lineHeight={2.5}
+                  align="center"
+                  tooltip={
+                    'Starts infinite(until you step/run out of things to craft) crafting items/food.'
```

Comment:
Mentioning that it gets interrupted when you move is redundant, given that's the standard behavior for how all crafting works. 

```suggestion
                    'Repeat this craft until you run out of ingredients.'
```